docs/concept-selection.md

```diff
@@ -61,24 +61,78 @@ formulating design requirements from there.
     - Learn technologies I can’t use at work, without the overhead of it being
       safe.
 
+## Objective Criteria
+### Complexity
 
-## Complexity
+Complex systems are more prone to design errors, have more failure modes, and
+lead to overengineered solutions. Simple or elegent solutions are always
+preferred over complicated ones, when possible. 
 
-To be determined.
+Necessary complexity should be targeted and limited in scope. See also
+[Difficulty](#difficulty).
 
-## Novelty
+### Cost
 
-To be determined.
+This project is self funded out of our own pockets. Naturally, a lower cost is
+better.
 
-## Difficulty
+### Potential for Future Work
 
-To be determined.
+Engineering hours have the most "bang-for-buck" value on work that can be reused
+in future projects, or at least contributes to technical skills of the team (see
+also [Personal Development](#personal-development)). Concepts that have a clear
+path for future work or use in a future project are preferred.
 
-## Cost
+### Modularity
+The ability for a concept to be broken down into smaller ones is an important
+characteristic that can balance out [Complexity](#complexity). What we look for
+is to get the most value out of every engineering hour spent working on this
+project. So if a concept can be decomposed into smaller, independently
+useful modules, those engineering hours don't go to waste. Even better if the
+modules have [Potential for Future Work](#potential-for-future-work).
 
-To be determined.
+No matter the [Complexity](#complexity) or [Difficulty](#difficulty), a
+modular concept is always preferred because it enables the best chance of
+success.
```

Comment:
@Beaudidly is this along the lines of what you were thinking?

Reply:
Kinda yeah, I don't think it captures my desire to have stopping points where we can launch with some MVP if we are way past our schedule.  Its not so much that it can be a module, but more that a complex/difficult module should have multiple tiers of success with one of them being the least we would settle for. 

Example of a system that while _modular_ that goes against what I'm looking for here:
"tlm module, avionics module, sensing module, nuclear fusion module"
Here, if the super complex module fails, we don't really have a payload or experiment of any sort.

Something tiered _and_ modular that I do like:
"tlm, sensing, avionics, single axis spinning w/o controls (maybe just gas + valves), single axis spin control (add more complexity), 3 axis controls, propulsive landing"  
Where maybe that first leap into spinning is a checkpoint where if we do that, we hit our minimum objective.  Lets say we hit the next one, single axis control.  But when striving for 3-axis control, we stall, but thats okay.  We launch with the last checkpoint we slapped.